docs/language/operators/summarize.md

```diff
@@ -34,7 +37,16 @@ to a cluster of workers in an adaptive shuffle, though this is not yet implement
 
 ### Examples
 
-Sum the input sequence:
+Average the input sequence:
+```mdtest-command
+echo '1 2 3 4' | zq -z 'summarize avg(this)' -
+```
+=>
+```mdtest-output
+{avg:2.5}
+```
+
+Sum the input sequence, leaving out the `summarize` keyword:
```

Comment:
I figured that since we used the explicit `summarize` in one example and call attention here to intentionally leaving it out, now we're free to leave it out of all the other examples.